pkg/api/types.go

```diff
@@ -195,6 +195,119 @@ type VolumeSource struct {
 	NFS *NFSVolumeSource `json:"nfs"`
 }
 
+// Similar to VolumeSource but meant for the administrator who creates PVs.
+// Exactly one of its members must be set.
+type PersistentVolumeSource struct {
+	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
+	// kubelet's host machine and then exposed to the pod.
+	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"gcePersistentDisk"`
+	// HostPath represents a directory on the host.
+	// This is useful for development and testing only.
+	// on-host storage is not supported in any way
+	HostPath *HostPathVolumeSource `json:"hostPath"`
+}
+
+type PersistentVolume struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	//Spec defines a persistent volume owned by the cluster
```

Comment:
In the fullness of time users will (with the blessing of clusteradmin) be able to make a claim and have a volume to back that claim be provisioned dynamically.

In the fullness of time users will (with the blessing of clusteradmin) be able to request a pod and have a node to back that pod be provisioned dynamically.

For a rough rule of thumb: if it costs the cluster admin money, user should not be able to do it unless explicitly allowed.  Consider on-prem.  Users are bound by the pool of hardware available - they might be able to auto-provision new hardware, but it involves the clusteradmin interm going to Fry's.